tests/testthat/test_featsel_filters.R

```diff
@@ -72,3 +72,16 @@ test_that("randomForestSRC_var.select filter handles user choices correctly", {
     more.args = list("randomForestSRC_var.select" = c(method = "vh.imp"))))
   )
 })
+
+
+test_that("Ensemble filters can accept duplicate base methods", {
```

Comment:
```suggestion
test_that("Ensemble filters can deal with non-unique base methods ", {
```

Reply:
If you like, but I think "accept duplicate" is clearer and more succinct.

Reply:
"duplicate" indicates that is indeed a full duplicate - which it is not, because the params are different. 

"non-unique base methods" indicates that some names/methods might not be unique in the selection but not necessarily a duplicate (due to different params).

Reply:
That's a good point. I will change it.